packages/react-ui-components/src/Dialog/style.css

```diff
@@ -63,6 +63,9 @@
 .dialog__actions > * {
     margin-left: 2px;
 }
+.dialog__closeBtn {
```

Comment:
Hey, just noticed this, please don't do such hacks.
Also, you are remove X close icon on all dialogs, not only on the insert node dialog. Is it intended?
I'm no UX expert, but I think some people might be used to it.

Reply:
Thank you, will have a look!

Reply:
Bring back the close button ;-) Maybe we can remove it later on if we really don't need it, but it does no harm if it is there.